app/views/downloads/download_windows.html.erb

```diff
@@ -10,12 +10,28 @@
   <h1>Downloading Git</h1>
   <div class="callout downloading">
     <h3>Your download is starting...</h3>
-    <p>
-      <%= raw "You are downloading the latest (<strong>#{@download.version.name}</strong>) <strong>#{@bitness}</strong> version of <strong>Git for Windows</strong>. This is the most recent <a href='#{@project_url}'>maintained build</a>. It was released <strong>#{time_ago_in_words @download.release_date} ago</strong>, on #{@download.release_date.strftime("%Y-%m-%d")}." %>
-    </p>
-    <p>
-      <%= raw "<strong>If your download hasn't started, <a href=\"#{@download.url}\">click here to download manually</a>.</strong>" %>
-    </p>
+    <script>
+        if (navigator.userAgent.search('WOW64|Win64') != -1) {
+            var version             = '<%= @download64.version.name %>';
+            var bitness             = '64-bit';
+            var time_ago_in_words   = '<%= time_ago_in_words @download64.release_date %>';
+            var release_date        = '<%= @download64.release_date.strftime("%Y-%m-%d") %>';
+            var download_url        = '<%= @download64.url %>';
+        } else {
+            var version             = '<%= @download32.version.name %>';
+            var bitness             = '32-bit';
+            var time_ago_in_words   = '<%= time_ago_in_words @download32.release_date %>';
+            var release_date        = '<%= @download32.release_date.strftime("%Y-%m-%d") %>';
+            var download_url        = '<%= @download32.url %>';
+        }
+
+        document.write(
```

Comment:
I'm not an expert on js, but from what I know, `document.write` is not recommended, and other approaches such DOM manipulation are preferred.

https://stackoverflow.com/questions/4537963/what-are-alternatives-to-document-write

If you have a good argument for using document.write (if you have a nice experience in JS and are sure about this), let us know, and I won't oppose :)

Reply:
I'm definitely not an expert. :)
But I suppose if we use DOM-manipulation - it can create a slight (may be even not noticeable) delay.

Reply:
i don't think the delay would be noticeable since it's just a small piece of logic. And having placeholders on the DOM + moving this code into a .js file, seems a cleaner approach than having these <script> tags in the middle of the HTML. 

since the issue is already fixed by the new cache rule (https://github.com/git/git-scm.com/issues/1129#issuecomment-363067019), I would prefer to have the clean approach.

Reply:
Hey @Dibro89, are you interested in trying the other approach? Otherwise I may try it by myself so that we can compare :)